### ISSUE TITLE
Regression in .pyxres tilemaps in version 1.5.3

### ISSUE THREAD

**Issue by danbst** [kitao/pyxel#348]
I've made a resource file for a game with Pyxel editor v1.5.0
It works fine in game.

However, when using this same pyxres file in Pyxel game or editor v1.5.3 I no longer see tilemaps drawn with `pyxel.bltm()`. It looks like regression.

Here is the pyxres file. I've changed extension to .txt to allow upload to Github. 
[labi.pyxres.txt](https://github.com/kitao/pyxel/files/7775435/labi.pyxres.txt)

Edit: here is game source code
<details>

```
import pyxel
import random
from dataclasses import dataclass
from enum import IntEnum, auto

START_LEVEL = 0

@dataclass
class Box:
    x:int
    y:int
    vx:float
    vy: float

class GameState(IntEnum):
    MENU = auto()
    LAUNCH = auto()
    DEATH = auto()
    WIN = auto()

def clamp(x, a, b):
    return max(a, min(b, x))

class MySuperGame:
    """ Шаблон для Pyxel проекту """
    state = GameState.LAUNCH

    def __init__(self):
        """ Цей код запуститься при старті """
        pyxel.init(128, 128, fps=30)
        pyxel.load('labi.pyxres')
        pyxel.playm(0, loop=True)

        self.x = 0
        self.y = 0
        self.vx = 0
        self.vy = 0
        self.moving = False
        self.watching = 8
        self.boxes = []
        self.level = 16*START_LEVEL

        pyxel.run(self.update, self.draw)

    def die(self):
        pyxel.load('labi.pyxres')
        #pyxel.stop()
        self.state = GameState.DEATH
        self.x, self.y = 1, 1
        self.boxes.clear()

    def update(self):
        """ Логіка, фізика, кнопки і клавіші оброблються тут """
        if self.state == GameState.LAUNCH:
            self.moving = False
            self.vx = 0
            self.vy = 0
            if pyxel.btn(pyxel.KEY_LEFT):
                self.vx = -2
                self.moving = True
            if pyxel.btn(pyxel.KEY_RIGHT):
                self.vx = 2
                self.moving = True
            if pyxel.btn(pyxel.KEY_UP):
                self.vy = -2
                self.moving = True
            if pyxel.btn(pyxel.KEY_DOWN):
                self.vy = 2
                self.moving = True
            self.x += self.vx
            self.y += self.vy
            for box in self.boxes:
                box.x += box.vx
                box.y += box.vy
                if not (0 <= box.x <= pyxel.width-8):
                    box.vx *= -1
                if not (0 <= box.y <= pyxel.height-8):
                    box.vy *= -1
                if abs(box.x - self.x) < 5 and abs(box.y - self.y) < 5:
                    self.die()
            self.x = clamp(self.x, 0, pyxel.width-8)
            self.y = clamp(self.y, 0, pyxel.height-8)

    def draw(self):
        """ Відмальовка екрану """
        pyxel.cls(0)
        if self.state == GameState.LAUNCH:
            fc = pyxel.frame_count
            # for i in range(16):
            #     pyxel.colors[i] = mx*123999 + my*345*i + i*10
            level = self.level
            pyxel.bltm(0, 0, 0, level, 0, 32, 32, 0)
            tm = pyxel.tilemap(0)
            for px in range(-2, 3):
                for py in range(-2, 3):
                    x, y = self.x//8+px, self.y//8+py
                    if tm.pget(level+x, y) == (1, 0):
                        tm.pset(level+x, y, (0,0))
                        self.boxes.append(Box(
                            self.x+px*8, self.y+py*8, 
                            vx=(random.uniform(0.5, 1)*random.choice([1,-1]))*1.5, 
                            vy=(random.uniform(0.5, 1)*random.choice([1,-1]))*1.5))
            if tm.pget(level+(self.x+4)//8, (self.y+4)//8) == (0, 1):
                self.state = GameState.WIN
                self.x, self.y = 1, 1
                self.boxes.clear()
                return
            for box in self.boxes:
                pyxel.blt(box.x, box.y, 0, 8, 0, 8, 8, 0)
            if self.moving:
                anim = [16, 32, 16, 24, 24][fc//5 % 5]
                if self.vx > 0:
                    pyxel.blt(self.x, self.y, 0, anim, 0, 8, 8, 0)
                else:
                    pyxel.blt(self.x, self.y, 0, anim, 0, -8, 8, 0)
            else:
                if random.random() < 0.1:
                    self.watching = -self.watching
                anim = self.watching
                pyxel.blt(self.x, self.y, 0, 16, 0, anim, 8, 0)
        elif self.state == GameState.DEATH:
            pyxel.text(30, 40, "   You died.\n\nDon't touch boxes", 8)
            if pyxel.btn(pyxel.KEY_RETURN):
                self.state = GameState.LAUNCH
        elif self.state == GameState.WIN:
            pyxel.text(30, 40, "   WINNER", pyxel.frame_count % 16)
            if pyxel.btn(pyxel.KEY_RETURN):
                self.state = GameState.LAUNCH
                self.level += 16
MySuperGame()
```
</details>

**Comment by kitao** [comment]
Those should be fixed in 1.5.4. Please try the latest version.
And please note that the current `bltm` requires positions in pixel not tile.
That made the API incompatible, but it's much easier to use.

**Comment by danbst** [comment]
Okay, I see. Now all args in `bltm` are pixels. Makes sense.
